README.md

```diff
@@ -1,113 +1,12 @@
-6. List installed servers with helm: helm list 
-7. Watch the server start in kubernetes by running: `kubectl get pods`
-8. Use the pod name you get from last command to follow the console logs: `kubectl logs -f iq-nexus-iq-server-xxx` 
-9. Confirm expected version numbers in those logs.
-10. Forward a localhost port to a port on the running pod: `kubectl port-forward iq-nexus-iq-server-xxx 8070`
-11. Connect and check that your fresh new server is successfully running: `http://localhost:8070/`
-12. Uninstall the server with helm: `helm delete iq` 
-13. Confirm it's gone: `helm list && kubectl get pods`
-14. Shutdown minikube: `minikube stop`
-
-### Running Unit Tests
-To unit test the helm charts you can follow the next steps:
-
-1. Install the unittest plugin for Helm: https://github.com/quintush/helm-unittest
-2. Run the tests for each individual chart:
-     * `cd charts/<deployment>/nexus-repository-manager; helm unittest -3 -t junit -o test-output.xml .`
-
-### Running Integration Tests
-You can run the integration tests for the helm charts by running the next commands. 
-
-Before running the integration tests:
-* Install docker, helm, kubectl, and [minikube](https://minikube.sigs.k8s.io/docs/start/), if you don't already have it on your local workstation.
-  * You could also use docker with k8s enabled instead of minikube.
-* The integration tests will be executed on a running cluster. Each test will create a new POD that will connect to the server installed by our 
-helm chart. Check [this](https://helm.sh/docs/topics/chart_tests/)
-
-
-Running integration tests for Nexus Repository Manager:
-1. From source code: `helm install nxrm ./charts/<deployment>/nexus-repository-manager --wait`
-3. Run the tests: `helm test nxrm`
-
-### Further Notes on Usage
-
-#### Resolver File and Ingress-DNS
-
-Get the default `values.yaml` for each chart.
-- Nexus Repository: `helm show values nexus-repo sonatype/nxrm3-helm-repository > iq-values.yaml`
-
-Edit the values file you just downloaded to enable ingress support, and install the chart 
-with those values:
-
-- Nexus Repository: `helm install nexus-repo sonatype/nxrm3-helm-repository -f repo-values.yaml`
-
-If you want to use the custom values file for the demo environment that expose 
-the apps on a local domain of *.demo which is done by creating a resolver file. 
-On a Mac it's `/etc/resolver/minikube-minikube-demo` with the following entries:
-```
-domain demo
-nameserver 192.168.64.8
-search_order 1
-timeout 5
-```
-
-You'll need to update the IP address to match the running instance's IP address.
-Use `minikube ip` to get the address
-
-Docs for Ingress-dns are here
-https://github.com/kubernetes/minikube/tree/master/deploy/addons/ingress-dns
+See the [Single-Instance OSS/Pro Kubernetes Chart](./https://github.com/sonatype/nxrm3-helm-repository/tree/main/single-inst-oss-pro-kubernetes) if you are doing the following:
```

Comment:
Perhaps we should also mention that this is for Orient deployments @vswaminathan777  @lisadurant ?

Reply:
yes, we can mention that this is for embedded OrientDB deployments

Reply:
@bobotimi @vswaminathan777 addressed!